crates/pgmq/tests/integration_test.rs

```diff
@@ -329,7 +329,7 @@ async fn test_database_error_modes() {
     // we expect a database error
     match queue {
         Err(e) => {
-            if let pgmq::errors::PgmqError::DatabaseError { .. } = e {
+            if let pgmq::errors::PgmqError::UrlParsingError { .. } = e {
```

Comment:
can you add a test that covers the `DatabaseError` case? I think it would look something like:
- provided a VALID database connection string, but with either wrong host or password, `pgmq::errors::PgmqError::DatabaseError` is returned.

Reply:
Keep the existing test that makes sure `DatabaseError` happens when its the wrong, but valid, connection string.

Add new test that validates that a poorly formed connection string returns `UrlParsingError`

Reply:
Yep, I'll add that